examples/message-ubench/main.pony

```diff
@@ -113,8 +115,12 @@ actor SyncLeader
   var _total_count: U64 = 0
   var _current_t: I64 = 0
   var _last_t: I64 = 0
+  var _run_done: Bool = false
+  var _timers: Timers tag
```

Comment:
this could be:

`let _timers = Timers`

Reply:
fixed